packages/ui/ToggleItem/ToggleItem.js

```diff
@@ -224,6 +287,7 @@ const ToggleItemBase = kind({
  */
 const ToggleItemDecorator = compose(
 	ForwardRef({prop: 'componentRef'}),
+	Slottable({slots: ['itemIconAfter', 'itemIconAfterText', 'itemIconBefore', 'itemIconBeforeText']}),
```

Comment:
I found use cases that using multiple `itemIcon`s in the latest UX scenario. So, I added 4 slots.

Reply:
The presence of these 4 slots implies that it's completely acceptable to have **all 4 icons** displaying on one ToggleItem at the same time, in addition to the built-in `ToggleIcon`. I would like to _strongly discourage_ this, as it is an unsustainable component design. If there is such a need for an enormous amount of icons on an item, we have either failed in designing the component API or the UX.

Reply:
That makes sense.
We have not seen more than 2 displays so far. (Maximum 2) So why do not we just select 2 `itemIconPosition`s and suggest using all use cases equally?

I also think the best way is that there is just 1 slot. And we can choose the position. (before, beforeText, after, afterText)